main_courses/medium/stuffed-peppers.md

```diff
@@ -0,0 +1,36 @@
+# Stuffed peppers with mushrooms
+
+This vegan recipe is sure to impress your guests and makes for a lovely main course!
+
+Adapted from [this recipe](https://www.lalena.ro/u/reteta/902/Ardei-umpluti-cu-ciuperci/) with slight alterations - check it out if you need visuals.
+
+Serves **8-11**
+
+## Ingredients
+
+* 11 peppers (I tend to use red or yellow)
+* 3 brown onions
+* 2 carrots
+* 300g Champignon mushrooms (any of the smaller mushrooms varieties should do)
+* 200g rice (risotto or generally white rice)
+* 1 bunch of fresh dill
+* 1 can chopped tomatoes
+* 1 tbsp dry thyme
+* 60ml oil
+* 1 tsp salt
+* 1 tsp pepper
+* 2 vegetable stock cubes
+
+## Method
+
+1. Wash the vegetables. Finely chop the onions and grate the carrots.
+2. Add oil to the pan along with the onions and the carrots, fry for 7-8 minutes.
+3. Finely chop the mushrooms.
+4. Add the rice and 100ml water to the pan with the onions and carrots.
+5. Add thyme and a bit of salt to the pan and let it cook for another 10 minutes.
+6. Prepare the peppers for stuffing. Depending on their size and how much you want to stuff them, you will need anywhere from 8 to 11 of them.
```

Comment:
This could probably be removed with adding the halving note to the `ingredients` section.